modules/op-validating-pull-requests-using-GitHub-interceptors.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="op-validating-pull-requests-using-GitHub-interceptors_{context}"]
+= Validating pull requests using GitHub Interceptors
+
+Using a GitHub Interceptor, you can validate processing of pull requests based on GitHub owners configured for a repository. The GitHub Interceptor processes a pull request only if the user name is listed as an owner or a comment is issued by an owner of the repository. This validation helps you to prevent unnecessary execution of a `PipelineRun` or `TaskRun` object.
```

Comment:
Can you also specify what happens when repo is private

like you mentioned in filtering part
ex: https://github.com/tektoncd/triggers/blob/main/examples/v1beta1/github-owners/github-eventlistener-interceptor.yaml#L22

Reply:
@savitaashture  Incorporated you suggestion. Please verify and confirm.